1_data_prep/12_create_train_val_split.py

```diff
@@ -364,13 +380,51 @@ def visualize_split_with_trees(plots_gdf, species_matrix, plot_weights, lump_lev
     plt.show()
 
 
+def exclude_withheld_test_plots():
+    """
+    Exclude datasets that belong to withheld test plots.
+
+    Returns:
+        pd.DataFrame: DataFrame of datasets belonging to train and val plots only.
+    """
+    renders_dir = path_config.rendered_instance_ids
+    withheld_plots_csv = path_config.withheld_plots_file
+
+    # Extract the names of all dataset folders which have renders
+    dataset_names = [
```

Comment:
Nice. I meant to ask if you'd updated the plots for this run, so this is a great way to do it.